controllers/vspherevm_controller.go

```diff
@@ -98,6 +92,7 @@ func AddVMControllerToManager(ctx context.Context, controllerManagerCtx *capvcon
 		// Watch the controlled, infrastructure resource.
 		For(controlledType).
 		WithOptions(options).
+		Named(controllerNameShort).
```

Comment:
```suggestion
```

Let's not set this here. Basically CR will use lower-case kind as a default. This aligns to what we do in core CAPI and other non-supervisor controllers. I only didn't change it for some supervisor controllers (I'll take a closer look at those in a follow-up)

Reply:
I have reverted this change, feel free to create a followup.

JFYI - there are other instances with named controllers that have been added recently.

Reply:
I'll follow-up